app/src/main/res/layout/beneficiary_spinner_layout.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
this xml file name should be descriptive_spinner_layout.xml

Reply:
@therajanmaurya  I used a similar naming convension as [File](https://github.com/openMF/mifos-mobile/blob/development/app/src/main/res/layout/account_spinner_layout.xml).